tests/logger_tests.cpp

```diff
@@ -43,6 +43,11 @@ TEST(Adaptor, factory) {
 TEST(Adaptor, EnviromentPath) {
   rmm::mr::cuda_memory_resource upstream;
 
+  unsetenv("RMM_LOG_FILE");
```

Comment:
Would it be worthwhile to query the variable first and reset it at the end instead of clobbering it with `envtest.txt` here?

Reply:
Probably.

Reply:
Ok, added an RAII blob to restore the value from before the test

Reply:
Nice!